shared-bindings/bleio/Peripheral.c

```diff
@@ -68,25 +72,27 @@ static const char default_name[] = "CIRCUITPY";
 //|
 //|    # Create a peripheral and start it up.
 //|    periph = bleio.Peripheral([service])
-//|    periph.start_advertising()
+//|    adv = ServerAdvertisement(periph)
+//|    periph.start_advertising(adv.advertising_data_bytes, adv.scan_response_bytes)
 //|
 //|    while not periph.connected:
 //|        # Wait for connection.
 //|        pass
 //|
-//| .. class:: Peripheral(services, *, name='CIRCUITPY')
+//| .. class:: Peripheral(services=(), \*, name='CIRCUITPY')
 //|
 //|   Create a new Peripheral object.
```

Comment:
How does this API change when we're always already a Peripheral to expose the UART service and file service? No need to change it in this PR but we should do it before 5.0.0 is stable.

It is a similar problem to how USB is started prior to the code.py VM. Theoretically the user code could advertise a second UART service for their own data.

Reply:
We talked about this on video chat. The NUS UART is inadequate for REPL use in, say, a classroom situation, because there is no pairing/bonding.

But see some relatively simple changes to NUS that support pairing: https://devzone.nordicsemi.com/nordic/nordic-blog/b/blog/posts/support-pairing-on-nus-profile-for-central-and-per.